src/main/scala/mesosphere/marathon/core/volume/Volumes.scala

```diff
@@ -0,0 +1,97 @@
+package mesosphere.marathon.core.volume
```

Comment:
I don't understand this comment. Can you elaborate?

On Mon, Mar 21, 2016 at 9:34 AM, Matthias Veit <notifications@github.com>
wrote:

> In src/main/scala/mesosphere/marathon/core/volume/Volumes.scala
> <https://github.com/mesosphere/marathon/pull/3525#discussion_r56821625>:
>
> > @@ -0,0 +1,97 @@
> > +package mesosphere.marathon.core.volume
>
> All packages under core are components that contain:
>
>    - interface definitions (traits)
>    - impl that holds private implementation
>    - Module that defines the required/provided interfaces
>
> We are moving the code from * to core/module thats why it is probably
> confusing.
> But moving Volumes here feels wrong.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/marathon/pull/3525/files/45e09ae46811bf8a3a0aa2ed8f1d09e9a69d7b71#r56821625>
>